### PR TITLE
Check OSS License in existing db

### PR DESCRIPTION
**Pull request by riyenas0925**
### License check target
* License (Unconfirmed license)
* License (The field is required)
* oss version unconfirmed (unconfirmed version)
* oss name unconfirmed (unconfirmed open source)
* License (declared)

#### Search in existing db (search priority)
1. find by oss name and oss version
2. find by oss download location and version
3. find by oss download location

### Preview
GIF 2021-11-10 오후 5-51-26

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by riyenas0925** [comment]
@soimkim 

## Test File
[TEST CASE 7.xlsx](https://github.com/fosslight/fosslight/files/7537045/TEST.CASE.7.xlsx)

### OSS List
image

### Comment
* **For testing, I added a version of xstream with the same license and download location.**

**Comment by riyenas0925** [comment]
@soimkim 
How can I change the text below?
image

**Comment by soimkim** [comment]
@riyenas0925, 
License detected based on OSS Name, Version, and Download location. To change the license, click the "Check License" button.
OSS Name, Version, Download location을 기반으로 검출된 License 입니다. License를 변경하시려면 Check License 버튼을 클릭하세요.

**Comment by riyenas0925** [comment]
@soimkim Changed the check license popup discription text

File: src/main/webapp/WEB-INF/views/admin/project/identification.jsp [hunk 1]
```diff
@@ -335,6 +335,9 @@
                     	</c:if>
                     	<c:if test="${project.dropYn ne 'Y' and (ct:isAdmin() or project.viewOnlyFlag eq 'N')}">
                     		<input type="button" value="Check OSS Name" onclick="com_fn.CheckOssViewPage('SRC')" class="btnColor red srcBtn" style="width: 115px;" />
+                    	</c:if>
```
Comment (soimkim) [inline src/main/webapp/WEB-INF/views/admin/project/identification.jsp:338]: How about deleting lines 338 and 339?

File: src/main/webapp/WEB-INF/views/admin/oss/checkOssLicensepopup.jsp [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+	<body>
+		<div id="loading_wrap_popup" class="loading" style="display:none;">
+			<div class="loadingBlind"></div>
+			<img src="/images/loading.gif" alt="loading" />
+		</div>
+		<div id="wrap" style="padding-top: 20px;">
+			<div  align="center" >
+				<div class="jqGridSet" style="overflow: auto; width: 98%; height: 500px;">
+					<div align="left" style="padding-bottom: 20px;">
+						<b>License detected based on OSS Name, Version, and Download location. To change the license, click the "Check License" button.</b>
```
Comment (soimkim) [inline src/main/webapp/WEB-INF/views/admin/oss/checkOssLicensepopup.jsp:246]: Next time, let's put this statement in messages.properties.

File: src/main/java/oss/fosslight/controller/OssController.java [hunk 3]
```diff
@@ -1173,6 +1175,80 @@ public String ossDetailView(HttpServletRequest req, HttpServletResponse res, @Mo
...
+				map = selfCheckService.getIdentificationGridList(paramBean);
+				
+				break;
+			case CoConstDef.CD_CHECK_OSS_NAME_IDENTIFICATION:
+				map = projectService.getIdentificationGridList(paramBean);
+				
+				break;
+		}		
+
+		// 중간 저장을 기능 대응을 위해 save시 유효성 체크를 data load시로 일괄 변경
```
Comment (soimkim) [inline src/main/java/oss/fosslight/controller/OssController.java:1209]: Let's change the comments to English later

**Review by soimkim — APPROVED** [review]

**Pull request merged**